### ISSUE TITLE
Provide a system table to show the status of ON CLUSTER queries.

### ISSUE THREAD

**Issue by alexey-milovidov** [yandex/ClickHouse#6614]

**Comment by amosbird** [comment]
It's a much better mechaism compared to current query with output interface. It should behave like mutation queries and allows client applications to either polling on this system table, or do sync operation.

**Comment by alexey-milovidov** [comment]
We already have `system.distributed_ddl_queue` that shows the status of the tasks on the current server.
But we don't have anything to check the status of the queries issued on the current server.

Is that correct?

**Comment by amosbird** [comment]
> But we don't have anything to check the status of the queries issued on the current server.

Exactly. And we don't have `SYSTEM` queries to interact with those queries.

**Comment by alexey-milovidov** [comment]
This: https://clickhouse.com/docs/operations/system-tables/distributed_ddl_queue
